tools/sdk-sync/tests/mock_e2e_test.rs

```diff
@@ -1,605 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-
```

Comment:
I removed this test since it seems to be more of a drag than a boost after `e2e_test.rs` was added. It takes a lot of time to update for changes and doesn't really catch a whole lot.